api/app.rb

```diff
@@ -32,11 +32,10 @@ class App < Roda
 
     r.on 'api' do
       rodauth.require_authentication
-      r.on 'wards' do
-        r.route 'wards'
-      end
+      r.multi_route
     end
   end
 end
 require_relative 'routes/wards'
+require_relative 'routes/bonds'
```

Comment:
weird, we already have a `require_relative 'routes/bond'` 

I think these 2 ruby files should be merged.

Reply:
Fully agree. But yesterday I wasn't as fresh as needed to achieve it. There are two use cases for the bonds api that need diferent authenticacion schema, so we have to consider whether to move rodauth in the routing tree, or making it conditional:
```ruby
rodauth unless "code that looks wether a UUID is present in URL"
```

Reply:
Understandable. :)

Reply:
I solved it and added unit test coverage to both bond and wards API